Lib/codeop.py

```diff
@@ -86,24 +87,12 @@ def _maybe_compile(compiler, source, filename, symbol):
     with warnings.catch_warnings():
         warnings.simplefilter("error")
 
-        code1 = err1 = err2 = None
         try:
-            code1 = compiler(source + "\n", filename, symbol)
+            code1 = compiler(source + "\n", filename, symbol, flags=ast.PyCF_ALLOW_INCOMPLETE_INPUT)
```

Comment:
`code1` is no longer used, and line needs folding
```suggestion
            compiler(source + "\n", filename, symbol,
                           flags=ast.PyCF_ALLOW_INCOMPLETE_INPUT)
```
I noticed today that this is the second compiler call.  When making this the first and only call, compound statements no longer require a terminating blank, so test_codeop fails on this:
```
  File "f:\dev\3x\Lib\test\test_codeop.py", line 144, in test_incomplete
    ai("if 9==3:\n   pass\nelse:\n   pass")
  File "f:\dev\3x\Lib\test\test_codeop.py", line 49, in assertIncomplete
    self.assertEqual(compile_command(str, symbol=symbol), None)
```
and test_idle fails in 3 places.  Interactively in IDLE, a compound statement is seen as complete and executes as soon as a minimal suite is entered.
```
>>> if 9==3:
...     pass
>>> 
```
Unless you change this, please document, at least here, that this mode can only be used after a 'normal' call raises SyntaxError.